src/telliot_feed_examples/reporters/interval.py

```diff
@@ -153,7 +153,9 @@ async def ensure_staked(self) -> Tuple[bool, ResponseStatus]:
                 return True, status
             else:
                 status.error = (
-                    "Unable to stake deposit: " + write_status.error
+                    "Unable to stake deposit: "
```

Comment:
Consider using `telliot_core.utils.response.error_status` to reduce code.
See Contract.write for example usage.

Reply:
Will do